src/sentry/receivers/onboarding.py

```diff
@@ -402,7 +403,8 @@ def record_member_joined(organization_id: int, organization_member_id: int, **kw
         },
     )
     if created or rows_affected:
-        try_mark_onboarding_complete(organization_id)
+        user: RpcUser = Organization.objects.get(id=organization_id).get_default_owner()
```

Comment:
I would use here `Organization.objects.get_from_cache` as it might speed this up a little bit if we already have organization cached.

Also, looking at our codebase, i see that we more often use `get_from_cache` than `get`